packages/sources/tradermade-test/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@chainlink/tradermade-test-adapter",
+  "version": "0.0.0",
+  "description": "Chainlink tradermade-test adapter.",
+  "keywords": [
+    "Chainlink",
+    "LINK",
+    "blockchain",
+    "oracle",
+    "tradermade-test"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "repository": {
+    "url": "https://github.com/smartcontractkit/external-adapters-js",
+    "type": "git"
+  },
+  "license": "MIT",
+  "scripts": {
+    "clean": "rm -rf dist && rm -f tsconfig.tsbuildinfo",
+    "prepack": "yarn build",
+    "build": "tsc -b",
+    "server": "node -e 'require(\"./index.js\").server()'",
+    "server:dist": "node -e 'require(\"./dist/index.js\").server()'",
+    "start": "yarn server:dist"
+  },
+  "dependencies": {
+    "@chainlink/external-adapter-framework": "0.23.2",
```

Comment:
Sorry to keep tacking things on but we got a new version of the fw out if you could bump this when you get the chance

Reply:
Done, Also tested this EA manually